README.md

```diff
@@ -1,52 +1,53 @@
-Ja, greppa i refseq_20190301_ncbiRefSeq efter transkriptet/genen för att lugna sinnet kanske att inte skriptet bara är dum-i-huvet, men det har den faktiskt inte varit än så länge. 
-Är det transkript som saknas skulle jag testa att ta bort versions-namnet i transkriptet, det efter "." alltså. Och kolla om det finns en äldre version eller nyare. Finns det en annan version skulle jag maila och fråga den som bad om panelen ifall det är ok att vi tar den existerande versionen istället, brukar vara OK.
-Är det gener som saknas skulle jag gå in här:
-https://www.genecards.org
-Och söka efter det angivna gennamnet som inte hittas, och där kolla i listor efter synonymer/deprecated namn och försöka reda ut ifall det namnet vi har är det nyare coola namnet eller om vi sitter på det gamla töntiga namnet. Står det angivna gennamnet som dom skickade högst upp på sidan så är det tyvärr vi som har det töntiga namnet. Då får vi greppa efter synonymerna i vår fil tills att vi får en vettig träff.
-Sen samlar man på sig modet...och lovar sig själv att ta en lång renande dusch i någon snar framtid och kör:
-./replace_names.sh gammalt_namn nytt_namn
-Det här har gjorts några gånger redan, kolla här i repot:
-/create_bedfile/changes_made
-Men om man är rädd att man förstört något så finns en backup i samma mapp. 
+**options:**
+`-e 2`  Adds two bases, to make sure splice sites are included as well. This is usually something that KG wants.
 
-Historiskt sett, och inte egentligen på ett superorganiserat sätt så har jag haft min outputmapp för att skapa insilico-filer här:
-/medstore/Development/WGS_validation/in_silico_panels/
+##### Using a list of genes
+The following command will provide introns too.
+`./create_bed.py -r refseq_20190301_ncbiRefSeq -g <gene list> -o <output folder>`
 
-Men nu tänker jag att vi ska spara bedfilerna i det här repot, eftersom filerna ändå är ganska små och det möjliggör versionshantering! FANTASTISKT JU. 
+**options**
+`-l yes`  Extract the longest transcript for all genes; use in case you want to exclude the introns. (Genes can have overlapping exons of different lengths, and this can be problematic when calculating coverage.)
 
-För att vara lite bättre organiserad har jag sparat alla genlistor/transkriptlistor som har skickats i detta googlesheet. "InSilicopaneler KK" & "InSilicopaneler KG".
+KK usually wants all the information they can get, allowing you to run the command without the `-l`flag.
 
-# coverage analysis (for in silico panel verification into WGS analysis) 
+
+##### The genes/transcripts can’t be found
+Sometimes genes or transcripts can not be found in `refseq_20190301_ncbiRefSeq`. It could be that the transcript has been updated since 2019-03-01 or that the gene name has been changed. If an element of the input list was not found, the script will output a file named notfound.txt. Should the gene/transcript not be present in `refseq_20190301_ncbiRefSeq`there are a few things you can try to solve it:
```

Comment:
Would like some information regarding hg38 as well. This uses a different gtf-file

Reply:
Detta lämnas till en ny PR enligt överenskommelse mellan Emma ch Alvar